packages/compat/src/compat-adapters/ember-font-awesome.ts

```diff
@@ -0,0 +1,16 @@
+import V1Addon from '../v1-addon';
+import { AddonMeta } from '@embroider/core';
+
+export default class EmberFontAwesome extends V1Addon {
+  get packageMeta(): Partial<AddonMeta> {
+    let meta = super.packageMeta || {};
+    meta['public-assets'] = {
+      'node_modules/font-awesome/fonts/FontAwesome.otf': '/fonts/FontAwesome.otf',
```

Comment:
wondering, should it be `os` specific path separators?

Reply:
Ah, I think you're right. Also, it feels wrong to list the specific files instead of copying the whole fonts folder but

1) I think you need to list files, you cannot list folders in `public-assets`
2) Not sure how to do that as I can't use `const fs = require('fs'); fs.readdirSync(...)` and other node libraries here, can I?

Reply:
I think `walkSync` from https://www.npmjs.com/package/walk-sync may work for this case

Reply:
I see pefixed `@fontawesome` packages like - https://www.npmjs.com/package/@fortawesome/free-solid-svg-icons
https://github.com/FortAwesome/Font-Awesome/tree/master/js-packages/%40fortawesome
should current compat adapter handle it?

Reply:
I pushed a commit with the `walkSync` improvement, thank you for the suggestion.

Regarding handling all the files in those packages, my understanding is that compat-adapters are looked up using the package name, so if we wanted to make them compatible with Embroider, too, then an adapter would have to be implemented for each of them.

I might be missing the idea, though.

Reply:
I'm also not sure about the build error. It says

```
packages/compat/src/compat-adapters/ember-font-awesome.ts:14:7 - error TS2532: Object is possibly 'undefined'.
14       meta['public-assets'][path] = `/fonts/${fileName}`;
         ~~~~~~~~~~~~~~~~~~~~~
```

but there's this block 2 lines above that:

```js
    if (!meta['public-assets']) {
      meta['public-assets'] = {};
    }
```

I'm quite the newb when it comes to TS, though, so it's probably me.